config/reference_container.yaml

```diff
@@ -0,0 +1,8 @@
+ 
+reference_container_folder: "/home/sven/Documents/reference_container/"
+reference_container_names:
+  - broad-hg38-bwaidx_v0
+  - ncbi-hg38-bwaidx_v0
+  - broad-hg38-variation_v0
+  - ncbi-t2t-chm13hg002_v0
+USE_REFCON: False
```

Comment:
this option (note: not all uppercase, that is the style inside the constants module; in the config, it should be lowercase) should have a more explicit name in the config, i.e. "use_reference_container" (true/false).

Reply:
I hope I addressed that issue appropriately in the latest commit